kolibri/plugins/facility/assets/src/modules/userManagement/handlers.js

```diff
@@ -1,22 +1,99 @@
 import pickBy from 'lodash/pickBy';
-import FacilityUserResource from 'kolibri-common/apiResources/FacilityUserResource';
 import samePageCheckGenerator from 'kolibri-common/utils/samePageCheckGenerator';
+import FacilityUserResource from 'kolibri-common/apiResources/FacilityUserResource';
 import { _userState } from '../mappers';
+
+export function fetchSortedFacilityUsersHandler(
+  store,
```

Comment:
With @BabyElias, we looked into `unknown local action type: handleApiError` and `unknown local action type: notLoading` and I think that had something to do with this `store` instance.  I've tried to adjust several places and was able to make a bit of progress but was running into some other things to resolve around global store access, and didn't manage to understand and fix fully in the time I had.

Taking into account we agreed to get WIP work as soon as possible in, and we resolved the other issue, I think this is not blocking and we can merge now. Perhaps @BabyElias and @nucleogenesis you can coordinate on opening follow-up issues and deciding on whom to assign.